packages/atomic/src/components/commerce/atomic-commerce-products-per-page/atomic-commerce-products-per-page.new.stories.tsx

```diff
@@ -0,0 +1,102 @@
+  title: 'Atomic-Commerce/ProductsPerPage',
+  id: 'atomic-commerce-products-per-page',
+
+  render: renderComponent,
+  decorators: [decorator],
+  parameters,
+  play: async (context) => {
+    await play(context);
+    await playExecuteFirstSearch(context);
+  },
+};
+
+export default meta;
+
+export const Default: Story = {
+  name: 'atomic-commerce-products-per-page',
+};
+
+export const InAPage: Story = {
+  name: 'In a page',
+  decorators: [
+    (story) =>
+      html` <atomic-commerce-layout>
+        <atomic-layout-section section="facets">
+          <atomic-commerce-facets></atomic-commerce-facets>
+        </atomic-layout-section>
+        <atomic-layout-section section="main">
+          <atomic-layout-section section="status">
+            <atomic-commerce-query-summary></atomic-commerce-query-summary>
+            <atomic-commerce-sort-dropdown></atomic-commerce-sort-dropdown>
+          </atomic-layout-section>
+          <atomic-layout-section section="products">
+            <atomic-commerce-product-list
+              display="grid"
+              density="compact"
+              image-size="small"
+            >
+            </atomic-commerce-product-list>
+            <atomic-commerce-query-error></atomic-commerce-query-error>
+          </atomic-layout-section>
+          <atomic-layout-section section="pagination">
+            ${story()}
+          </atomic-layout-section>
+        </atomic-layout-section>
+      </atomic-commerce-layout>`,
+  ],
+  play: async (context) => {
+    await play(context);
+    await playExecuteFirstSearch(context);
+  },
+};
+
+export const InAPageWithCustomChoicesDisplayed: Story = {
+  name: 'In a page with custom choices displayed',
+  args: {
+    'attributes-choices-displayed': '2,3,4,5,6',
+  },
+  tags: ['test'],
+  decorators: [
+    (story) =>
+      html` <atomic-commerce-layout>
+        <atomic-layout-section section="facets">
+          <atomic-commerce-facets></atomic-commerce-facets>
+        </atomic-layout-section>
+        <atomic-layout-section section="main">
+          <atomic-layout-section section="status">
+            <atomic-commerce-query-summary></atomic-commerce-query-summary>
+            <atomic-commerce-sort-dropdown></atomic-commerce-sort-dropdown>
+          </atomic-layout-section>
+          <atomic-layout-section section="products">
+            <atomic-commerce-product-list
+              display="grid"
+              density="compact"
+              image-size="small"
+            >
+            </atomic-commerce-product-list>
+            <atomic-commerce-query-error></atomic-commerce-query-error>
+          </atomic-layout-section>
+          <atomic-layout-section section="pagination">
+            ${story()}
+          </atomic-layout-section>
+        </atomic-layout-section>
+      </atomic-commerce-layout>`,
+  ],
+  play: async (context) => {
+    await play(context);
+    await playExecuteFirstSearch(context);
+  },
+};
```

Comment:
This is annoying but because of this regex in Storybook via the `buildArgsParam` in our `load` function, I was unable to sent via the url the choicesDisplayed attribute since it contains commas `,`. Since I couldnt just send a attribute via the url, I had to create a whole story for it.
https://github.com/storybookjs/storybook/blob/4f9108cc05621d913aff9163243c3af8ba80b9f2/code/core/src/router/utils.ts#L66